### DISCUSSION TITLE
How do I tell how complete the data is?

### DISCUSSION THREAD

**Discussion by VincentLivant** [freelawproject/courtlistener#1611] [Q&A]
Hello,

I'm a student working on a CS410 project proposal at University of Massachusetts Boston.  I was wondering if you could help me to determine how extensive the coverage of opinions relating to tax law is for our project proposal.  Particularly, I am having trouble determining if more data might exist in PACER or other sources that I might not have access to through your API.  Is there a way for users to determine this?

Very respectfully,
Vinny Livant

**Comment by mlissner** [comment]
Vincent, our tax coverage is quite good, and we have a post about it here: https://free.law/2020/03/03/free-law-project-now-has-every-reported-us-tax-opinion/. 

But when it comes to whether there might be something of use in PACER, it's more difficult to reply. There's an immense amount of content in PACER, much of which *nobody* has. Some of our PACER APIs are not publicly accessible without chatting with us first, but all of our tax opinions are in the Opinions DB where anybody can access them via an API. 

Does that help?

**Reply by VincentLivant** [reply]
Thanks for the reply and informative link!  This helps a lot, especially with my understanding of why this question seemed so difficult to answer.

**Comment by flooie** [comment]
Hey @VincentLivant 

You'll need to elaborate a bit on the types of opinions you are referring to as well.  We have collected all of the precedential (ie important) decisions dating back to the 1940's that came from the Board of Tax Appeals and later the Federal Tax Courts.  

The federal district courts clearly can make rulings on tax law and those would be in PACER but that is much more subjective.
